src/Microsoft.ComponentDetection.Orchestrator/Experiments/Models/ExperimentDiff.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace Microsoft.ComponentDetection.Orchestrator.Experiments.Models;
+
+using System.Collections.Generic;
+using System.Linq;
+
+/// <summary>
+/// A model for the difference between two sets of <see cref="ExperimentComponent"/> instances.
+/// </summary>
+public class ExperimentDiff
+{
+    /// <summary>
+    /// Creates a new <see cref="ExperimentDiff"/>.
+    /// </summary>
+    /// <param name="oldComponents">A set of components from the control group.</param>
+    /// <param name="newComponents">A set of components from the experimental group.</param>
+    public ExperimentDiff(
+        IEnumerable<ExperimentComponent> oldComponents,
+        IEnumerable<ExperimentComponent> newComponents)
```

Comment:
nit: use the same terminology all the way through. You switch between old/new and control/experimental.